src/doc/manifest.md

```diff
@@ -128,6 +128,9 @@ git = "https://github.com/wycats/hammer.rs"
 
 [dependencies.color]
 git = "https://github.com/bjz/color-rs"
+
+[dependencies.geometry]
+path = "crates/geometry"
```

Comment:
While you're at it, could you change these dependencies to the more concise inline table syntax?